### PR TITLE
[4.0] Fix moving a list item on hover

### PR DESCRIPTION
**Pull request by srishty-07**
Pull Request for Issue #33798.

### Testing Instructions
Go to systemm->global-config->plugin->permission

on hover, there is a jump of menu.

### Actual result BEFORE applying this Pull Request

https://user-images.githubusercontent.com/70388131/118277062-fff9bf00-b4e5-11eb-84c5-365ce8931c13.mp4

### Expected result AFTER applying this Pull Request
srishty

### CODE REVIEW

**Comment by richard67** [comment]
@srishty-07 The SCSS style linter in drone complains here: https://ci.joomla.org/joomla/joomla-cms/43676/1/20

```
administrator/templates/atum/scss/vendor/joomla-custom-elements/joomla-tab.scss
 142:9  ✖  Expected "border-top" to come before "box-shadow"   order/properties-order
```

File: administrator/templates/atum/scss/vendor/joomla-custom-elements/joomla-tab.scss [hunk 1]
```diff
@@ -138,7 +138,9 @@ joomla-tab {
         background-color: var(--atum-bg-dark-60);
         background-image: none;
         border-right: 0;
-        box-shadow: none;
+        border-top: 1px solid transparent;
+        border-bottom: 1px solid #dee2e6;
+        box-shadow: none;
```
Comment (richard67) [inline administrator/templates/atum/scss/vendor/joomla-custom-elements/joomla-tab.scss]:
```suggestion
        box-shadow: none;
```
Now the linter complained about unexpected white space at the end of this line.

File: administrator/templates/atum/scss/vendor/joomla-custom-elements/joomla-tab.scss [hunk 2]
```diff
@@ -137,7 +137,9 @@ joomla-tab {
         color: var(--atum-text-light);
         background-color: var(--atum-bg-dark-60);
         background-image: none;
-        border-right: 0;
+        border-top: 1px solid transparent;
+        border-right: 0;
```
Comment (richard67) [inline administrator/templates/atum/scss/vendor/joomla-custom-elements/joomla-tab.scss]:
```suggestion
        border-right: 0;
```
:-)

**Comment by srishty-07** [comment]
Thank you @richard67

**Comment by brianteeman** [comment]
image

**Comment by Kostelano** [comment]
Maybe remove it in joomla-tab.css?

```
joomla-tab > ul li + li {
margin-left: 1px;
}
```

Screenshot_1

**Comment by brianteeman** [comment]
Whatever you do and wherever you do it take extra care that it doesn't break orientation=horizontal

**Comment by srishty-07** [comment]
sir,
According to me, The reason for the jump in the menu is because of the border top and bottom given to the anchor tag and the default height of all li’s are the same, so due to that border there is a jump on hover, 
Screenshot (69)_LI

so I added border-top and bottom as it was there on ‘a’, on hover to prevent jumping, but I am not able to manage the height of the right side border on hover, it is taking the by default height of the li.
hover

And space as shown in the picture is coming because of the transparent border-top that is given to it on hover.

**Comment by ceford** [comment]
I have tested this item :white_check_mark: successfully on 069d22e8f2d8c0a5bf0b547c198aa1af3fae0877<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33871">issues.joomla.org/tracker/joomla-cms/33871</a>.</sub>

**Comment by himanshu007-creator** [comment]
I have tested this item :white_check_mark: successfully on 069d22e8f2d8c0a5bf0b547c198aa1af3fae0877<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33871">issues.joomla.org/tracker/joomla-cms/33871</a>.</sub>

**Comment by richard67** [comment]
@srishty-07 Could you address here the issue @brianteeman discovered? Or is that a completely different thing?

**Comment by srishty-07** [comment]
> sir,
> According to me, The reason for the jump in the menu is because of the border top and bottom given to the anchor tag and the default height of all li’s are the same, so due to that border there is a jump on hover,
> Screenshot (69)_LI
> 
> so I added border-top and bottom as it was there on ‘a’, on hover to prevent jumping, but I am not able to manage the height of the right side border on hover, it is taking the by default height of the li.
> hover
> 
> And space as shown in the picture is coming because of the transparent border-top that is given to it on hover.

.. @richard67  @brianteeman

**Comment by drmenzelit** [comment]
@srishty-07 I did a check of the scss code, if you remove line 59 
` &[active],
  &:hover {
      border: 0;
`
and line 131 
` a[role=tab] {
....
    border-top: 1px solid transparent;
`
you get rid of the jump and the glitch found by Brian.

Please test it and check if the changes have side effects on other parts of the backend.

**Comment by srishty-07** [comment]
> @srishty-07 I did a check of the scss code, if you remove line 59
> `&[active], &:hover { border: 0;`
> and line 131
> `a[role=tab] { .... border-top: 1px solid transparent;`
> you get rid of the jump and the glitch found by Brian.
> 
> Please test it and check if the changes have side effects on other parts of the backend.

Sure ma'am will try the changes.

**Comment by Quy** [comment]
Closing in favor of #34813. Thanks.

**Pull request closed without merging**